Brown-coder.js

```diff
@@ -0,0 +1,5 @@
+let fullName= "Adesuwa Adolosa";
+let id= "HNG-02362";
+let lang= "Javascript";
+let email = "adesuwahealthalive@gmail.com";
+console.log(`Hello World, this is ${fullName} with HNG ID: ${id} and email address: ${email}, using ${lang} for stage 2 task` )
```

Comment:
hey, @Brown-coder, edit your console log output to follow this ; 
**Hello World, this is [full name] with HNGi7 ID [ID] and email [email] using [language] for stage 2 task**
remove the full colons in your output as they are not part of the guided output ':'

Reply:
thanks